src/main/resources/view/MainWindow.fxml

```diff
@@ -33,11 +33,12 @@
           </Menu>
         </MenuBar>
 
-        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
           <padding>
-            <Insets top="5" right="10" bottom="5" left="10" />
+            <Insets top="10" right="10" bottom="10" left="10" />
           </padding>
-        </StackPane>
+          <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+        </VBox>
```

Comment:
Leaving a note here for future GUI reference.
Swapped positions of `personListPanelPlaceholder` with `commandBoxPlaceholder`.